lib/events.js

```diff
@@ -87,6 +87,7 @@ const {
 
 const kCapture = Symbol('kCapture');
 const kErrorMonitor = Symbol('events.errorMonitor');
+const kShapeMode = Symbol('shapeMode');
```

Comment:
Ideally this wouldn't be an extra property and share a bit field with captureRejections and maybe _maxListeners, but that doesn't need to happen in this PR